nodejs/week1/warmup/warmup.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="warmup.js"></script>
+</body>
+</html>
```

Comment:
This is not necessary. You can run the warmup.js file directly from node. 

`node warmup.js`